lib/guardian/permissions/bitwise_encoding.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Guardian.Permissions.BitwiseEncoding do
+  @behaviour Guardian.Permissions.PermissionEncoding
+  use Bitwise
+
+  def encode(value, _type, _perm_set) when is_integer(value) do
+    value
+  end
+
+  def encode(value, type, perm_set) when is_list(value) do
+    perms = Map.get(perm_set, type)
+    Enum.reduce(value, 0, &encode_value(&1, perms, &2))
+  end
+
+  defp encode_value(value, perm_set, acc),
+    do: perm_set |> Map.get(to_string(value)) |> bor(acc)
```

Comment:
I would prefer to have `do ... end` since it makes it harder to read, no a big deal whichever way.